ansible/configs/base-infra/requirements.yml

```diff
@@ -0,0 +1,13 @@
+---
+collections:
```

Comment:
Actually this may require conversation as well.

Just like with common packages I think we should have two collections. One for the config and one additional one (extra_collections?) for specific roles?

Reply:
Simplified to basics